templates/mempoolTx.html

```diff
@@ -0,0 +1,122 @@
+{{ define "js" }}
+{{ end }}
+
+{{ define "css" }}
+  <style></style>
+{{ end }}
+
+{{ define "content" }}
+  {{ with .Data }}
+    <div class="container mt-2">
+      <div class="my-3">
+        <div class="d-md-flex py-2 justify-content-md-between">
+          <h1 class="h4 mb-1 mb-md-0">
+            <span class="ml-1 mr-1"><i class="fas fa-credit-card mr-2"></i>Pending Transaction Details</span>
+          </h1>
+          <nav aria-label="breadcrumb">
+            <ol class="breadcrumb font-size-1 mb-0" style="padding: 0; background-color: transparent;">
+              <li class="breadcrumb-item"><a href="/" title="Home">Home</a></li>
+              <li class="breadcrumb-item active" aria-current="page">Mempool Tx Details</li>
+            </ol>
+          </nav>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card-header">
+          <ul class="nav nav-tabs card-header-tabs border-bottom-0" role="tablist">
+            <li class="nav-item">
+              <a class="nav-link show active" id="overview-tab" data-toggle="tab" href="#overview" role="tab" aria-controls="overview" aria-selected="true"><i class="fas fa-info-circle"></i><span class="tab-text" style="margin-left: 6px;">Overview</span></a>
+            </li>
+          </ul>
+        </div>
+        <div class="card-body px-0 py-1">
+          <div class="tab-content h-100">
+            <div id="overview" class="tab-pane d-flex flex-column fade show active" role="tabpanel" aria-labelledby="overview-tab">
+              <div class="d-flex flex-column p-3">
+                <div class="col-md-3"><span class="align-middle">Transaction Hash:</span></div>
```

Comment:
The field `Transaction Hash:` has its value below the name and not to the right of it like done for everything else:
![image](https://user-images.githubusercontent.com/114066135/222080731-3031ef6a-23e2-4047-8b5a-81dd4f70c0d2.png)

Furthermore it is missing the thin line below it.

Please change.

Reply:
fixed